calm/draft/2024-12/meta/core.json

```diff
@@ -175,6 +181,7 @@
     "node-type-definition": {
       "enum": [
         "actor",
+        "ecosystem",
```

Comment:
I'm not sure what this is for? Will continue looking through the review.

Reply:
Explained on linked issue. If we don't want to pull over that terminology, happy to go with something else